source/common/runtime/runtime_features.cc

```diff
@@ -131,5 +135,25 @@ RuntimeFeatures::RuntimeFeatures() {
   }
 }
 
+bool RuntimeFeatures::enabledByDefault(absl::string_view feature) const {
+  if (feature.substr(0, ConstantQuicFlagPrefix.length()) == ConstantQuicFlagPrefix) {
+    auto res = quiche::FlagRegistry::getInstance().findFlag(absl::StrCat(
```

Comment:
It seems like it might be nice if we could move some of this logic into the FlagsRegistry itself. I think that both the ConstantQuicFlagPrefix and "FLAGS_quic_reloadable_flag_" logic could all be hidden inside of the registry. What do you think?

Reply:
Absolutely! done.